global.json.bak

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file should not exist.   Do not make changes outside of the /sdk/servicebus area